src/main/java/ch/njol/skript/variables/Variables.java

```diff
@@ -346,14 +346,27 @@ public static Object getVariable(final String name, final @Nullable Event e, fin
 			}
 		}
 	}
-	
+
+	/**
+	 * Deletes a variable.
+	 * 
+	 * @param name The variable's name.
```

Comment:
```suggestion
	 * @param name The variable's name. Can be a "list variable::*" (<tt>value</tt> must be <tt>null</tt> in this case)
```
Do you think it's worth including this part? Taken from javadoc of setVariable method below.

Reply:
No because the value is always going to be null until this gets changed in the future with a variable api update rather than null for deletion.

Reply:
Oh I had misread it - my bad!